indico/util/date_time.py

```diff
@@ -149,6 +149,9 @@ def format_interval(start_dt, end_dt, format='yMd', locale=None):
     if not locale:
         locale = get_current_locale()
 
+    if locale and format == 'small':
+        format = 'hm' if 'a' in locale.time_formats['short'].pattern else 'Hm'
```

Comment:
This feels kind of weird... @tomasr8 can you have a look if there's a better option?

Reply:
Instead of checking the hour cycle and modifying the skeleton based on that, can we just pass the skeleton directly? i.e.
```suggestion
        # not sure where the skeleton is stored, just an example
        format = locale.time_formats['short'].skeleton
```